Sources/Internal/FrameProvider.swift

```diff
@@ -77,6 +79,16 @@ final class FrameProvider {
       let x = minXOfMonth(containingItemWithFrame: layoutItem.frame, at: position)
       let y = minYOfMonth(containingDayItemWithFrame: layoutItem.frame, atRowInMonth: rowInMonth)
       return CGPoint(x: x, y: y)
+        
+    case .monthFooter(let month):
+      let x = layoutItem.frame.origin.x
+        let y = layoutItem.frame.minY -
+            monthHeaderHeight -
+            content.monthDayInsets.top -
+            (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) -
+            CGFloat((CGFloat(numberOfRows(in: month)) * daySize.height) + (CGFloat(numberOfRows(in: month) - 1) * content.verticalDayMargin)) -
+            content.monthDayInsets.bottom
```

Comment:
Similar feedback as here https://github.com/airbnb/HorizonCalendar/pull/75/files#r533051542

Reply:
Be careful of the sign though, this one is subtracting parts of the layout, while the linked comment is adding parts of the layout. It'll be pretty obvious if it's wrong though when you build and run ;)

Reply:
This is all working as expected now.
I will spend some time on implementing some form of footer experience that I could see worthy of this.
Do you want me to solely focus on UI presentation of a footer, or actually the UI and logic implementation?

Can see it getting quite intense just for a demo that is all, I'm indifferent, just want some direction on how thought out I should be in the demo.